account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -544,13 +654,21 @@ def _find_taxes(self):
                 # Check the tax for changes
                 tax = self.env['account.tax'].browse(tax_id)
                 notes = self.diff_notes(template, tax)
+
+                missing_xml_id = False
+                if self.recreate_xml_ids:
+                    missing_xml_id = self.missing_xml_id(tax)
```

Comment:
Use `get_external_id` in all occurrences. Can we refactor this to call to the same method for all objects for not repeating so many code?

Reply:
I've refactored to not repeat so much code, but I didn't use here get_external_id, I think it would get a little complicated if I have to use it here